index.js

```diff
@@ -26,8 +26,24 @@ maths.import({
 
 function transformResolve(value) {
     return reduceFunctionCall(value, 'resolve', function(argString) {
+
+        var unit = '';
+        if (argString.indexOf('floor(') >= 0
```

Comment:
Looks good, but I wonder if we could move this logic and adapt the `Unit` prototype instead maybe? This'll mean we can support multiple floor/ceiling calls in one resolve string. Open to your preference :+1: